fcm/atmosphere/_mars_atm_api.py

```diff
@@ -32,41 +32,59 @@ def martian_atmosphere_api(latitude, longitude, timestamp):
         
     timestamp : Union[datetime.date, datetime.datetime]
         timestamp for which to request the data
-    
+
+    zkey : int
```

Comment:
Missing type hint here that it's an optional parameter:
```python
zkey : int, optional
```

Reply:
done